foreshadow/concrete/internals/cleaners/json_flattener.py

```diff
@@ -62,3 +64,25 @@ class StandardJsonFlattener(BaseCleaner):
     def __init__(self):
         transformations = [json_flatten]
         super().__init__(transformations)
+
+    def metric_score(self, X: pd.DataFrame) -> float:
```

Comment:
This is not related to this code change. But please open issues to track:
1. Potential performance improvement on applying cleaner transformation using Series.apply() or DataFrame.apply() instead iterate through rows.
2. Look at the metric calculation to understand whether the matched_len from the cleaner transformation function should be between 0 and 1.